tests/ChangeLog

```diff
@@ -34,6 +34,11 @@
 	  or if --enable-debug was specified during configure
 	* testsuite.src: adjusted several tests to use that option
 
+2024-02-26  Boris Eng <boris.eng@ocamlpro.com>
+
+	* testsuite.at: added a new test suite to test the backward compatibility
+	  of strings functions (INSPECT, STRING, UNSTRING)
```

Comment:
that note is valid, please add a new changelog entry that says something aling "drop content as this only starts to be relevant after the first 4.0 release" and replace its contents by a m4 macro to say the same